Improve clarity on `.has` Map method (#20697)
mdn/content: files/en-us/web/javascript/reference/global_objects/map/index.md

  - : Returns the value associated to the `key`, or `undefined` if there is
    none.
- {{jsxref("Map.prototype.has()")}}

>   - : Returns a boolean asserting whether a value has been associated to a passed
>     `key` in the `Map` object or not.

Reviewer: ```suggestion
  - : Returns a boolean indicating whether a value has been associated with a passed `key` in the `Map` object or not.
```
